src/views/components/tooltip/testid-tooltip/TestIdTooltip.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { Dimensions } from 'react-native-web';
+import { string, node } from 'prop-types';
+import Tooltip from '../Tooltip';
+
+class TestIdTooltip extends Component {
+  static defaultProps = {
+    id: 'test-id',
+  };
+
+  static propTypes = {
+    id: string,
+    children: node,
+  };
+
+  handleMouseOver = () => {
+    // eslint-disable-next-line no-console
+    console.warn( 'TestId:', this.props.id );
+  };
+
+  render() {
+    const { id, children } = this.props;
+
+    const dimensions = Dimensions.get( 'window' );
+
+    let isDebugMode = window.originalQueryParams.showcodes;
+
+    if (
+      localStorage.getItem( 'showcodes' ) === true ||
```

Comment:
The use of `localStorage` here will also break on Native. In other areas of code we're using the `Storage` util we created that works between Native and Web.

[/src/utils/storage@`master`](https://github.com/genny-project/alyson-v3/tree/master/src/utils/storage)

We should change the usage of `localStorage` here to `Storage` (`/src/utils/storage`).

Additionally, be wary that sometimes when storage is set, it is stringified. Meaning, if we're doing a triple eq check (`===`) for `true`, it mightn't work. Be wary!